Deprecaited value in Mac OS

```
# github.com/anatol/smart.go
nvme_darwin.cpp:35:54: warning: 'kIOMasterPortDefault' is deprecated: first deprecated in macOS 12.0 [-Wdeprecated-declarations]
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/IOKit.framework/Headers/IOKitLib.h:133:19: note: 'kIOMasterPortDefault' has been explicitly marked deprecated here
nvme_darwin.cpp:36:50: warning: 'kIOMasterPortDefault' is deprecated: first deprecated in macOS 12.0 [-Wdeprecated-declarations]
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/IOKit.framework/Headers/IOKitLib.h:133:19: note: 'kIOMasterPortDefault' has been explicitly marked deprecated here
```

--- a/nvme_darwin.cpp
+++ b/nvme_darwin.cpp
@@ -32,8 +32,8 @@ unsigned int smart_nvme_open_darwin(const char *path, void **ptr) {
   IOCFPlugInInterface **plugin;
   struct smart_nvme_darwin *nvme;
 
-  CFMutableDictionaryRef matcher = IOBSDNameMatching(kIOMasterPortDefault, 0, path);
-  io_object_t disk = IOServiceGetMatchingService(kIOMasterPortDefault, matcher);
+  CFMutableDictionaryRef matcher = IOBSDNameMatching(kIOMainPortDefault, 0, path);
+  io_object_t disk = IOServiceGetMatchingService(kIOMainPortDefault, matcher);
 
   while (!is_smart_capable(disk)) {
     io_object_t prevdisk = disk;

